include/OpenColorIO/OpenColorIO.h

```diff
@@ -1000,7 +995,7 @@ class OCIOEXPORT Config
     /**
      * Most applications will use the preceding method, but this method may be
      * used for applications that want to know which was the highest priority rule to match
-     * filePath.  The :cpp:func:`FileRules::getNumCustomKeys` and custom keys methods
+     * filePath. \ref FileRules::getNumCustomKeys and custom keys methods
```

Comment:
So is it correct that Doxygen finds the class references but it needs some extra help for this reference?  Could you share an HTML file that shows/confirms how the linking will work?

Reply:
To be honest, I'm still wrapping my head around references, and they're the most WIP-y, drafty part of this PR.

For certain, the `:cpp:func:` stuff is only getting in the way (it renders as text), and I'm pretty sure the `\ref` isn't actually doing anything useful itself. Doxygen itself knows to automatically link the text `FileRules::getNumCustomKeys` in this case, which is in fact what's happening in the [presently-live docs](https://opencolorio.readthedocs.io/en/latest/api/config.html#_CPPv4NK16OpenColorIO_v2_06Config25getColorSpaceFromFilepathEPKcR6size_t). Doxygen will likewise do its best to any other classes / enums / functions it recognizes. 

I'm pretty sure adding the `\ref` isn't helping anything, but it doesn't show up in the rendered HTML, and it seems benign (although I do need to check if need to check if causes warnings). I figured it might help Michael's python doc stuff differentiate between references and plain old words. And to be perfectly clear, this particular change doesn't change the linking behavior on the cpp side, it just takes out the `:cpp:func:` business, and removes the backtick-style  formatting (which we may want to keep).

(Sidebar: In retrospect, it might maybe it's better to just use backticks instead of using `\ref ` as a convention for indicating to Michael's Python docs generation stuff which words should be considered a link)

However, there's something else happening with this particular example -- it's an overloaded function, but the python side doesn't seem to recognize it as such; it's only rendering the documentation for the previously-defined function, despite having a different signature. I'm not sure why this is happening, because overloaded functions are handled fine [in other cases](https://opencolorio.readthedocs.io/en/latest/api/config.html#PyOpenColorIO.Config.GetProcessorFromConfigs).